src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -600,14 +614,20 @@ public boolean shouldExpire()
 
         long lastActivity
                 = packetIOActivity.getLastOverallActivityTimestampMs();
+        long now = System.currentTimeMillis();
         if (lastActivity <= 0)
         {
+            Duration timeSinceCreation = Duration.ofMillis(now - creationTimeMillis);
+            if (timeSinceCreation.compareTo(EP_TIMEOUT) > 0) {
+                logger.info("Endpoint's ICE connection has neither failed nor connected " +
+                    "after " + timeSinceCreation + ", expiring");
```

Comment:
`Duration.toString()` output looks like `PT2M3.345S` for values in this range. Is this what we want?

Reply:
i agree it's not the most legible, but i actually prefer it to reading a big number of milliseconds.  or is there a way to better format duration's output?

Reply:
Yeah, it's better than a big number of ms.

Barring creating our own formatters for `Duration` (which we could do but is overkill for this) this should be good enough.